packages/client/test/test-utils/utils.ts

```diff
@@ -223,3 +226,23 @@ export const waitForCalls = async (mockFunction: jest.Mock<any>, n: number): Pro
         return `Timeout while waiting for calls: got ${mockFunction.mock.calls.length} out of ${n}`
     })
 }
+
+export const startTestServer = async (
```

Comment:
It could be nice to have this function as part of `test-utils` package. But up to you to decide whether or not it is worth the move.